src/permutation/permutation.rs

```diff
@@ -998,13 +999,13 @@ mod test {
         /*
         Check that the unique encodings of the sigma polynomials have been computed properly
         Left_Sigma : {R0,O1,R2,O0}
-            When encoded using w, K1,K2,K3 we have {1 * K1, w * K2, w^2 *K1, w^3 * K2}
+            When encoded using w, K1,K2,K3 we have {1 * K1, w * K2, w^2 * K1, 1 * K2}
         Right_Sigma : {R1, O2, O3, L0}
-            When encoded using w, K1,K2,K3 we have {1 * K1, w * K2, w^2 * K2, w^3}
+            When encoded using w, K1,K2,K3 we have {w * K1, w^2 * K2, w^3 * K2, 1}
```

Comment:
The change is fine but it may be worth noting, that this is a semantic. 1 is technically correct but w is better. just that `BlsScalar::one` represents the first root of unity. So initial element for w group is one.

Reply:
I think it's worth changing, for:
R0 -> 1 * K1
R1 -> w * K1
L0 -> 1

Reply:
I agree